controllers/storagecluster/reconcile.go

```diff
@@ -415,6 +415,12 @@ func (r *StorageClusterReconciler) reconcilePhases(
 		return reconcile.Result{}, err
 	}
 
+	if instance.Spec.ExternalStorage.Enable {
```

Comment:
can we move to  above 
`objs = []resourceManager{`

Reply:
Actually i have kept it separate so that the lists logic wont get clutter.